server/src/internalClusterTest/java/org/elasticsearch/search/query/MultiMatchQueryIT.java

```diff
@@ -630,147 +623,87 @@ public void testCrossFieldMode() throws ExecutionException, InterruptedException
                         .lenient(true)
                         .analyzer("category")
                 )
             ),
-            response -> assertFirstHit(response, hasId("theone"))
-        );
-
-        assertResponse(
             prepareSearch("test").setQuery(
                 randomizeType(multiMatchQuery("15", "skill").type(MultiMatchQueryBuilder.Type.CROSS_FIELDS).analyzer("category"))
             ),
-            response -> assertFirstHit(response, hasId("theone"))
-        );
-
-        assertResponse(
             prepareSearch("test").setQuery(
                 randomizeType(multiMatchQuery("25 15", "skill").type(MultiMatchQueryBuilder.Type.CROSS_FIELDS).analyzer("category"))
             ),
-            response -> assertFirstHit(response, hasId("theone"))
-        );
-
-        assertResponse(
             prepareSearch("test").setQuery(
                 randomizeType(
                     multiMatchQuery("25 15", "int-field", "skill").type(MultiMatchQueryBuilder.Type.CROSS_FIELDS).analyzer("category")
                 )
             ),
-            response -> assertFirstHit(response, hasId("theone"))
-        );
-
-        assertResponse(
             prepareSearch("test").setQuery(
                 randomizeType(
                     multiMatchQuery("25 15", "first_name", "int-field", "skill").type(MultiMatchQueryBuilder.Type.CROSS_FIELDS)
                         .analyzer("category")
                 )
             ),
-            response -> assertFirstHit(response, hasId("theone"))
-        );
-
-        assertResponse(
             prepareSearch("test").setQuery(
                 randomizeType(
                     multiMatchQuery("25 15", "int-field", "skill", "first_name").type(MultiMatchQueryBuilder.Type.CROSS_FIELDS)
                         .analyzer("category")
                 )
             ),
-            response -> assertFirstHit(response, hasId("theone"))
-        );
-
-        assertResponse(
             prepareSearch("test").setQuery(
                 randomizeType(
                     multiMatchQuery("25 15", "int-field", "first_name", "skill").type(MultiMatchQueryBuilder.Type.CROSS_FIELDS)
                         .analyzer("category")
                 )
             ),
-            response -> assertFirstHit(response, hasId("theone"))
-        );
-
-        assertResponse(
             prepareSearch("test").setQuery(
                 randomizeType(
                     multiMatchQuery("captain america marvel hero", "first_name", "last_name", "category").type(
                         MultiMatchQueryBuilder.Type.CROSS_FIELDS
                     ).analyzer("category").operator(Operator.OR)
                 )
-            ),
-            response -> assertFirstHit(response, hasId("theone"))
+            )
         );
 
-        // test group based on analyzer -- all fields are grouped into a cross field search
-        assertResponse(
```

Comment:
I think that this one is missing too from the refactoring?

Reply:
Thank you Pano!

Reply:
Added back